src/sites/index.ts

```diff
@@ -17,6 +17,12 @@ export const Sites: Record<SiteId, Site> = {
 		paths: ['/home', '/compose/tweet'],
 		origins: ['http://twitter.com/*', 'https://twitter.com/*'],
 	},
+	hackernews: {
+		label: 'HackerNews',
```

Comment:
Could we change this a little so it doesn't scare the average user who might misinterpret 'Hacker':

```suggestion
		label: 'Y Combinator News (HN)',
```